### PR TITLE
New table "post-content" 

### PR DESCRIPTION
**Pull request by annando**
The table `item-content` is replaced with the table `post-content`.

### CODE REVIEW

File: database.sql [hunk 1]
```diff
@@ -797,55 +797,6 @@ CREATE TABLE IF NOT EXISTS `item` (
...
-	`uri` varchar(255) COMMENT '',
-	`uri-id` int unsigned COMMENT 'Id of the item-uri table entry that contains the item uri',
-	`uri-hash` varchar(80) NOT NULL DEFAULT '' COMMENT 'RIPEMD-128 hash from uri',
-	`activity` smallint unsigned NOT NULL DEFAULT 0 COMMENT '',
-	 PRIMARY KEY(`id`),
-	 UNIQUE INDEX `uri-hash` (`uri-hash`),
-	 INDEX `uri` (`uri`(191)),
-	 INDEX `uri-id` (`uri-id`),
-	FOREIGN KEY (`uri-id`) REFERENCES `item-uri` (`id`) ON UPDATE RESTRICT ON DELETE CASCADE
-) DEFAULT COLLATE utf8mb4_general_ci COMMENT='Activities for items';
```
Comment (MrPetovan) [inline database.sql:814]: Is this deletion intended?

Comment (annando) [reply database.sql:814]: Yeah. This table was unused for some time now.

**Review by MrPetovan — APPROVED** [review]

**Comment by AlfredSK** [comment]
Are the following warnings expected and normal or a reason to be concerned?

```
2021-01-31T11:04:39Z index [WARNING]: No post-content stored, quitting {"guid":"8d0ca5c7-6ba8648eae727ce7-bb0a073e","uri-id":3124828,"causer-id":0,"post-type":69,"network":"apub"} - {"file":"Item.php","line":1058,"function":"insert","uid":"580630","process_id":643720}
2021-01-31T11:05:46Z index [WARNING]: No post-content stored, quitting {"guid":"2ce9cc5045e20139ecfd005056264835","uri-id":3124833,"causer-id":0,"post-type":69,"network":"dspr"} - {"file":"Item.php","line":1058,"function":"insert","uid":"560c65","process_id":643716}
2021-01-31T11:10:11Z index [WARNING]: No post-content stored, quitting {"guid":"ccc77c6045e20139ed07005056264835","uri-id":3124840,"causer-id":0,"post-type":69,"network":"dspr"} - {"file":"Item.php","line":1058,"function":"insert","uid":"216b99","process_id":793466}
2021-01-31T11:10:12Z index [WARNING]: No post-content stored, quitting {"guid":"76158462c493febe086167b58df8ce1b3ff5aeeb","uri-id":3124841,"causer-id":126179,"post-type":64,"network":"apub"} - {"file":"Item.php","line":1058,"function":"insert","uid":"9674b6","process_id":793095}
```

**Comment by annando** [comment]
@AlfredSK yeah, just saw this in my logs as well.

Fixed by https://github.com/friendica/friendica/pull/9894

**Comment by AlfredSK** [comment]
@annando So you don't need the possible DB errors from log anymore? (I got a mail notification but it looks like you've edited your comment later)

**Comment by annando** [comment]
@AlfredSK exactly.

**Comment by AlfredSK** [comment]
I'm a little bit afraid git-pulling this one on my main node. But then again, who wants to live forever. ;-)

**Comment by annando** [comment]
Well, when you wait some time longer, you will have two for one. Means: two new tables in a single pull. :grimacing:

**Pull request merged**